cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/registry/AbstractTicketRegistry.java

```diff
@@ -38,6 +42,26 @@
     @Qualifier("ticketCipherExecutor")
     private CipherExecutor<byte[], byte[]> cipherExecutor;
 
+    @Nullable
+    @Autowired(required = false)
+    @Qualifier("ticketDelegators")
+    private List<Pair<Class<? extends Ticket>, Constructor<? extends AbstractTicketDelegator>>> ticketDelegators = new ArrayList<>();
```

Comment:
I don't think they are inconsistent, but practically, I agree they certainly won't be used at the same time.

To be consistent with other components in CAS, we offer this injection capability which is thus not required and we need to add the default service ticket delegators of course.

For the OAuth support, I could have created a specific `LinkedHashMap` in the OAuth Spring context XML file to inject it (though I choose another option by getting the ticket registry and adding the new service ticket delegators at the first positions).